examples/common/example_common.c

```diff
@@ -336,7 +346,8 @@ esp_loader_error_t load_ram_binary(const uint8_t *bin)
     printf("Start loading\n");
     esp_loader_error_t err;
     const esp_loader_bin_header_t *header = (const esp_loader_bin_header_t *)bin;
-    esp_loader_bin_segment_t segments[header->segments];
+    //esp_loader_bin_segment_t segments[header->segments];
```

Comment:
Why is this being moved to the heap?

Reply:
Because it didn't compile on MSVC

Reply:
A simple static array with a max segment count should suffice.